clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerPartitionAssignor.java

```diff
@@ -132,6 +132,16 @@ public void setGroupInstanceId(Optional<String> groupInstanceId) {
         public Optional<String> groupInstanceId() {
             return groupInstanceId;
         }
+
+        @Override
+        public String toString() {
+            return "Subscription(" +
+                "topics=" + topics +
+                ", userData=" + userData +
```

Comment:
The type of `userData` is `ByteBuffer` so it makes no sense to print it by `toString`. Maybe we can print the size?

Reply:
Oh, I didn't notice it! I logged the remaining size of the userData like the `Assignment.toString()` did. Thank you.